src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,35 +52,38 @@ public String todayDate(DateTimePart datePart) {
      * Return Optional of a date built from these elements.
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        if (dateParams.length == 3 && dateParams[1] >= 1 && dateParams[1] <= 12) {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        }
         return Optional.empty();
```

Comment:
you need to surround return with try and catch DateTimeParseException

Reply:
half of agree. not in this case. But I rewrote all the method and there is with catch